hugo.toml

```diff
@@ -22,10 +22,62 @@ title = 'MaxMind'
     name = 'minFraud'
     pageRef = '/minfraud'
     weight = 10
+
   [[menus.geoip]]
     name = 'GeoIP'
     pageRef = '/geoip'
     weight = 10
+    [[menus.geoip]]
+      name = 'Geolocate an IP'
+      pageRef = '/geoip/geolocate-an-ip'
+      parent = 'GeoIP'
+      weight = 10
+      [[menus.geoip]]
+        name = 'Databases'
+        pageRef = '/geoip/geolocate-an-ip/databases'
+        parent = 'Geolocate an IP'
+        weight = 10
```

Comment:
I still don't think I'm understanding the `weight` attribute here. How can `/geoip` `/geoip/geolocate-an-ip` and `/geoip/geolocate-an-ip/databases` all have the same weight number?

Reply:
From what I understand, these menu items have a different `parent` and therefore weights are reset as you descend the tree.

From https://gohugo.io/content-management/menus/
> weight
(int) A non-zero integer indicating the entry’s position relative the root of the menu, **or to its parent for a child entry**. Lighter entries float to the top, while heavier entries sink to the bottom.